src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/EServicesApiServiceImpl.scala

```diff
@@ -885,13 +885,16 @@ final case class EServicesApiServiceImpl(
             )
             .getOrElse(Json.Null),
           "docs"                    -> Json.arr(
-            descriptor.docs.map(doc =>
+            descriptor.docs.zipWithIndex.map { case (doc, index) =>
+              val extensionIndex = doc.name.lastIndexOf('.')
+              val newName        = if (index > 0) {
+                s"${doc.name.substring(0, extensionIndex)}-$index${doc.name.substring(extensionIndex)}"
```

Comment:
> having the documents
> 
>     * a.pdf
> 
>     * b.pdf
> 
>     * c.docx
> 
> 
> this would generate
> 
>     * a.pdf
> 
>     * b-1.pdf
> 
>     * c-2.docx
> 
> 
> is this the expected result?
> 
> Plus, I suggest to create a function and use it in both places

Yes, I thought it would be ugly to rename it if there are no conflict